unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCSocialApiBridge.cs

```diff
@@ -113,7 +113,7 @@ private async UniTask InitializeMatrixTokenThenRetrieveAllFriends(CancellationTo
 
             await foreach (var friends in friendsStream.WithCancellation(cancellationToken))
             {
-                tasks.AddRange(friends.Users_.Select(async friend =>
+                tasks.AddRange(friends.Users.Users_.Select(async friend =>
```

Comment:
https://github.com/decentraland/protocol/commit/9a568b16b2eafb134177329ba670c1451be8a169 This was merged but it didn't have its coding here.